File: C_Programming/simple_calculator.c
//Simple Calculator

#include <stdio.h>

int main() {

	char operator;
	double num1, num2;

	printf("Enter an operator (+, -, *, /): ");
	scanf("%c", &operator);

	printf("\nEnter two operands: ");
	scanf("%lf %lf", &num1, &num2);
	printf("\n");

	switch(operator) {

		case '+': 
				printf("Addition of %.2lf and %.2lf = %.2lf", num1, num2, num1 + num2);
				break;

		case '-': 
				printf("Subtraction of %.2lf and %.2lf = %.2lf", num1, num2, num1 - num2);
				break;

		case '*': 
				printf("Multiplication of %.2lf and %.2lf = %.2lf", num1, num2, num1 * num2);
				break;

		case '/': 
				printf("Division of %.2lf and %.2lf = %.2lf", num1, num2, num1 / num2);
				break;

		default: 
				printf("Please, enter valid operator.");
	}

	return 0;
}

File: Python/simple_calculator.py
#Simple Calculator

operator = input("Enter an Operator(+, -, *, /): ")
operand1, operand2 = map(int, input("Enter Two operands: ").split())

if operator == '+':
	print("Addition of {} and {} = {}".format(operand1, operand2, operand1 + operand2))

elif operator == '-':
	print("Subtraction of {} from {} = {}".format(operand1, operand2, operand1 - operand2))

elif operator == '*':
	print("Multiplication of {} and {} = {}".format(operand1, operand2, operand1 * operand2))

elif operator == '/':
	try:
		print("Division of {} by {} = {}".format(operand1, operand2, operand1 / operand2))
	except:
		print("Division by Zero")

else:
	print("Please, Enter valid operator.")